fe1-web/src/core/network/jsonrpc/messages/__tests__/MessageRegistry.test.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/dot-notation */
```

Comment:
why do you need to disable this ? are you trying to bypass the `private` field protection ? 

If so, a typical pattern is to create a `TestableMessageRegistry` which inherits from `MessageRegistry` and would somehow expose the ~~private~~ protected field. It makes the intention a tiny bit clearer.

Reply:
Given that `afterProcessingHandlers` are removed anyway, this problem is solved as well :)
But yes, I agree. But it is not straight forward to do so, since I actually want to test the `MessageRegistry` and not some `TestableMessageRegistry`. Extending the class should work though 🤔